apps/aareg/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/inntekt/HentArbeidsforholdRiverTest.kt

```diff
@@ -122,16 +123,18 @@ class HentArbeidsforholdRiverTest :
 private object Mock {
     fun innkommendeMelding(): HentArbeidsforholdMelding {
         val fnr = Fnr.genererGyldig()
+        val svarKafkaKey = KafkaKey(fnr)
 
         return HentArbeidsforholdMelding(
             eventName = EventName.AKTIVE_ORGNR_REQUESTED,
             behovType = BehovType.HENT_ARBEIDSFORHOLD,
             transaksjonId = UUID.randomUUID(),
             data =
                 mapOf(
+                    Key.SVAR_KAFKA_KEY to svarKafkaKey.toJson(),
```

Comment:
Kanskje ikke helt relatert til denne PR, men er planen at svarKafkaKey bare trenger å være i ett felt per melding? Det virker jo litt masete at den ligger både i data og utenfor

Reply:
På meldingen som publiseres på Redis så ligger det kun et sted. I meldingklassene, som her, så ligger det både i `data` og i `svarKafkaKey`. Det er vanlig for riverne som tar med all data de fikk inn når de publiserer (som vil si de fleste av riverne).

Reply:
Men er det egentlig nødvendig? Dette er jo ikke "data" per se, bare meta-info, litt som event og behov. Kan man ikke nøye seg med å ha det bare på ett sted?

Reply:
Jeg tenker det hører mer hjemme under data. Eventnavn, behov og kontekst-ID, som ligger på rot, er definerende for riveren. Eventnavn og behov identifiserer flyten, og kontekst-ID-en identifiserer konteksten (captain obvious her 🙋‍♂️). Disse verdiene blir brukt i alle rivers (eventnavn og behov sitt ansvarsområde er litt sammenblandet). `svarKafkaKey` er derimot noe data som skal brukes under utføring av hva enn riveren gjør. Det definerer ikke noe. Så da kan det likegodt slenges sammen med resten av dataen som brukes på samme måte.